commands/push.js

```diff
@@ -1,77 +1,95 @@
-'use strict';
+const cli = require('heroku-cli-util')
+const Sanbashi = require('../lib/sanbashi')
 
-const cli = require('heroku-cli-util');
-const co = require('co');
-const child = require('child_process');
+let usage = `
+    ${cli.color.bold.underline.magenta('Usage:')}
+    ${ cli.color.cmd('heroku container:push web')}         	           # Pushes Dockerfile in current directory to web process type
+    ${ cli.color.cmd('heroku container:push web worker --recursive')}     # Pushes Dockerfile.web and Dockerfile.worker found in current & subdirectories
```

Comment:
Thanks for the feedback -- @KarateCowboy, can we update with this text?  It adds new help text to `-R` and new text for the `heroku container:push` examples

https://docs.google.com/document/d/1DbuvumF3aerFqO-1h0W0evuVLFmV9XOp5bEXEnvSi3Y/edit#heading=h.kme0kogsywhu